jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/StepsRunnerIntegrationTest.java

```diff
@@ -63,8 +64,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** Integration tests for {@link BuildSteps}. */
-public class BuildStepsIntegrationTest {
+/** Integration tests for {@link StepsRunnerFactory}. */
+public class StepsRunnerIntegrationTest {
```

Comment:
Do you think it would be possible to consolidate a bit by turning this into `ContainerizerIntegrationTest` and moving the `StepsRunnerFactory` stuff into `Containerizer`, by just building the `StepsRunner`s within each of the `to` overloads?

Reply:
Ah, if we look at this from a different angle and think of this as `ContainerizerIntegrationTest`, then we may be able to build `StepsRunner` in `Conatinerizer.to()` as before. Let me try that.

Reply:
I think it worked out nicely.